pytest_pootle/fixtures/views.py

```diff
@@ -132,11 +135,21 @@ def project_views(request, client):
 
 
 @pytest.fixture(params=TP_VIEW_TESTS.keys())
-def tp_views(request, client):
+def tp_view_test_names(request):
+    return request.param
+
+
+@pytest.fixture(params=(None, 'member', 'admin'))
```

Comment:
Thanks for sharing the rationale. I only realize now that `tp_views` itself is another fixture — still, adding the parameters via `params=...` is definitely possible, and will feel more natural. I can make a PR if you are happy with it.